src/agent/srcview/src/bin/srcview.rs

```diff
@@ -132,7 +132,18 @@ fn srcloc(opts: SrcLocOpt) -> Result<()> {
     for modoff in &modoffs {
         print!(" +{:04x} ", modoff.offset);
         match srcview.modoff(modoff) {
-            Some(srcloc) => println!("{srcloc}"),
+            Some(locs) => {
+                let mut first = true;
+                for line in locs {
+                    if first {
+                        first = false;
+                    } else {
+                        print!(", ");
+                    }
+                    print!("{line}");
+                }
```

Comment:
Suggest that this is a little cleaner with `enumerate`:

```suggestion
                for (ix, line) in locs.enumerate() {
                    if ix > 0 {
                        print!(", ");
                    }
                    print!("{line}");
                }
```